### PR TITLE
Add FR-NL-2.03

### PR DESCRIPTION
**Pull request by allenbross-wf**
#### Reason for change
Added  FR-NL-2.04

#### Steps to Test

CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/plugin/validate/NL/rules/fr_nl.py [hunk 1]
```diff
@@ -163,6 +163,35 @@ def rule_fr_nl_1_01(
...
+        **kwargs: Any,
+) -> Iterable[Validation] | None:
+    """
+    FR-NL-2.03: The language of a report MUST be included in the 'xml:lang' attribute of the 'xbrli:xbrl' root element
+    """
+    modelXbrl = val.modelXbrl
+    for doc in modelXbrl.urlDocs.values():
+        if doc.type == ModelDocument.Type.INSTANCE:
+            lang = doc.xmlRootElement.get('{http://www.w3.org/XML/1998/namespace}lang')
+            if not lang or lang == "":
```
Comment (aaroncameron-wk) [inline arelle/plugin/validate/NL/rules/fr_nl.py:187]: Nit: `not lang` will already evaluate to `True` for an empty string
```suggestion
            if not lang:
```

File: arelle/plugin/validate/NL/rules/fr_nl.py [hunk 2]
```diff
@@ -194,6 +223,8 @@ def rule_fr_nl_2_04(
         )
 
 
+
+
```
Comment (aaroncameron-wk) [inline arelle/plugin/validate/NL/rules/fr_nl.py:227]: Nit: Unnecessary new lines

File: tests/resources/conformance_suites/nl_nt16/fr_nl/2-03-testcase.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      The instance root note is missing a value for 'xml:lang'.
+    </description>
+    <data>
+      <instance readMeFirst="true">2-03-invalid-lang.xbrl</instance>
+    </data>
+    <result>
+      <error>NL.FR-NL-2.03</error>
+    </result>
+  </variation>
+</testcase>
```
Comment (aaroncameron-wk) [inline tests/resources/conformance_suites/nl_nt16/fr_nl/2-03-testcase.xml:22]: Nit: newline (and in nt17, nt18)

**Review by aaroncameron-wk — APPROVED** [review]

**Review by austinmatherne-wk — DISMISSED** [review]

File: tests/resources/conformance_suites/nl_nt16/fr_nl/2-03-testcase.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        description="The language of a report MUST be included in the 'xml:lang' attribute of the 'xbrli:xbrl' root element."
+        outpath=''
+        owner="support@arelle.org"
+        xsi:schemaLocation="http://xbrl.org/2005/conformance https://www.xbrl.org/2005/conformance.xsd">
+  <variation id="missing-lang" name="Missing Language">
+    <description>
+      The instance root note is missing a value for 'xml:lang'.
+    </description>
+    <data>
+      <instance readMeFirst="true">2-03-invalid-lang.xbrl</instance>
```
Comment (aaroncameron-wk) [inline tests/resources/conformance_suites/nl_nt16/fr_nl/2-03-testcase.xml:16]: In each of the testcase files:
```suggestion
      <instance readMeFirst="true">2-03-missing-lang.xbrl</instance>
```

**Review by aaroncameron-wk — APPROVED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 Testing locally succeeded.

**Pull request merged**